bot/exts/moderation/incidents.py

```diff
@@ -178,6 +231,10 @@ class Incidents(Cog):
     Please refer to function docstrings for implementation details.
     """
 
+    # This dictionary maps a incident message to the message link embeds(s) sent by it
+    # RedisCache[discord.Message.id, List[discord.Message.id]]
+    message_link_embeds_cache = RedisCache()
```

Comment:
The type hint is incorrect, the value is just a string, delimited with commas. It'd probably be better explained using natural language; the hints don't really work.